index.css

```diff
@@ -1,23 +1,23 @@
-.panel#static {
```

Comment:
I did this to be extra cautious about duplicate elements, but see it is not really typical in css since the id should be unique document wide. Although browsers will probably apply the style to both elements since they are very forgiving. Thanks for the change.